<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>502</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>821</width>
    <height>502</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>821</width>
    <height>502</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">OSClicker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>iamges/icon.png</normaloff>iamges/icon.png</iconset>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string>Advanced autoclicker</string>
  </property>
  <property name="accessibleName">
   <string>OSClicker</string>
  </property>
  <property name="accessibleDescription">
   <string>Advanced AutoClicker with AntiBan Features</string>
  </property>
  <property name="iconSize">
   <size>
    <width>26</width>
    <height>26</height>
   </size>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif Collection</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>380</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif Collection</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Start (F8)</string>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>801</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="mainGroup">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="hotkeySettings">
        <property name="font">
         <font>
          <family>Sans Serif Collection</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Hotkeys</string>
        </property>
        <widget class="SingleKeySequenceEdit" name="triggerKey">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>100</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="keySequence">
          <string notr="true">F8</string>
         </property>
        </widget>
        <widget class="QLabel" name="hotkeyLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>60</y>
           <width>125</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Start/Stop Hotkey</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="customHotkey">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>30</y>
           <width>131</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Custom Hotkey</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="clickSettings">
        <property name="font">
         <font>
          <family>Sans Serif Collection</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Click Options</string>
        </property>
        <widget class="QSpinBox" name="clickAmount">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>40</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
        <widget class="QLabel" name="clickAmountL">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>40</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Click Amount</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="setClicks">
         <property name="geometry">
          <rect>
           <x>31</x>
           <y>46</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Set Clicks</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="infiniteClick">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>80</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Click until stopped</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="intervalSettings">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif Collection</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Click Interval</string>
     </property>
     <widget class="QSpinBox" name="hours">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="minutes">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="seconds">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="milliseconds">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
     <widget class="QLabel" name="hourLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Hours</string>
      </property>
     </widget>
     <widget class="QLabel" name="minutesLabel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Minutes</string>
      </property>
     </widget>
     <widget class="QLabel" name="secondsLabel">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Seconds</string>
      </property>
     </widget>
     <widget class="QLabel" name="msLabel">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Milliseconds</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="antibanSettings">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>259</y>
       <width>801</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif Collection</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Anti-Ban</string>
     </property>
     <widget class="QCheckBox" name="moveClick">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Move mouse with click</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="randomIntervals">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Random Intervals</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="intv1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="intv2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>49</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="intv3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>671</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
     <widget class="QLabel" name="intv1l">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Interval 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="intv2l">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Interval 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="intv3l">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Interval 3</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="windowMenu">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionMinimize"/>
    <addaction name="separator"/>
    <addaction name="actionPin"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionFAQ"/>
   </widget>
   <addaction name="windowMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
  </action>
  <action name="actionPin">
   <property name="text">
    <string>Pin on top</string>
   </property>
  </action>
  <action name="actionFAQ">
   <property name="text">
    <string>FAQ</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SingleKeySequenceEdit</class>
   <extends>QKeySequenceEdit</extends>
   <header>singlesequence</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
